pkgs/development/python-modules/kaitaistruct/default.nix

```diff
@@ -9,6 +23,15 @@ buildPythonPackage rec {
     sha256 = "3d5845817ec8a4d5504379cc11bd570b038850ee49c4580bc0998c8fb1d327ad";
   };
 
+  preBuild = lib.optionalString withCompress ''
+    ln -s ${kaitai_compress}/python/kaitai kaitai
+    sed '28ipackages = kaitai/compress' -i setup.cfg
+  '';
+
+  propagatedBuildInputs = lib.optionals withCompress [
+    lz4
+  ];
+
   meta = with lib; {
     description = "Kaitai Struct: runtime library for Python";
     homepage = "https://github.com/kaitai-io/kaitai_struct_python_runtime";
```

Comment:
Maybe you want to add the maintainers team here, too.

Reply:
Yeah, probably should since it is otherwise not maintained.